eli5/sklearn/transform.py

```diff
@@ -0,0 +1,51 @@
+"""transform_feature_names implementations for scikit-learn transformers
+"""
+
+from sklearn.pipeline import Pipeline, FeatureUnion
+from sklearn.feature_selection.base import SelectorMixin
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.preprocessing import Imputer
+
+from eli5.transform import transform_feature_names
+
+
+@transform_feature_names.register(Pipeline)
+def _pipeline_names(est, in_names=None):
+    names = in_names
+    for name, trans in est.steps:
+        if trans is not None:
```

Comment:
Changes released in scikit-learn 0.18 (scikit-learn/scikit-learn#1769) allow parts of a pipeline or union to be replaced or disabled in a grid search.